file-attribute-caching/build.gradle.kts

```diff
@@ -0,0 +1,131 @@
+plugins {
+    kotlin("jvm")
+    `maven-publish`
+    signing
+    alias(libs.plugins.ksp)
+}
+
+val attributeCachingFilesystemVersion: String by project
+version = attributeCachingFilesystemVersion
+
+ksp {
+    arg("autoserviceKsp.verify", "true")
```

Comment:
Is this actually used?

Reply:
yeah its how FileAttributeCachingFileSystemProvider.kt references the base FileSystemProvider and marius inserted that originally with his changes to get my code working